hazelcast/src/test/java/com/hazelcast/internal/util/phonehome/RESTClientPhoneHomeTest.java

```diff
@@ -149,4 +152,20 @@ public void queueOperations() throws IOException {
         );
     }
 
+    @Test
+    public void configUpdateOperations() throws IOException {
+        http.configReload(config.getClusterName(), "");
+        http.configUpdate(config.getClusterName(), "", "hazelcast:\n");
+
+        PhoneHome phoneHome = new PhoneHome(getNode(instance), "http://localhost:8080/ping");
+        phoneHome.phoneHome(false);
+
+        verify(1, postRequestedFor(urlPathEqualTo("/ping"))
+                .withRequestBody(containingParam("restconfigreloadsucc", "0"))
+                .withRequestBody(containingParam("restconfigreloadfail", "1"))
+                .withRequestBody(containingParam("restconfigupdatesucc", "0"))
```

Comment:
Why is this a 0? Is `"hazelcast:\n"` an invalid yaml config?

Reply:
these requests fail because config update/reload is EE feature. The success update/reload scenario is tested in EE PR